introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,797 @@
+{
```

Comment:
<p>Don't we need to delete the endpoint config and model too? Or does the <code>predictor</code> do this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
endpoint config and model don't actually use any resources (other than storage which is relatively cheap) and are typically not deleted. But, adding model deletion in this and all the subsequent notebooks to be extra cost efficient.